### DISCUSSION TITLE
ISPC on MinGW64 ?

### DISCUSSION THREAD

**Discussion by zx-82** [ispc/ispc#2356] [General]
Is anyone sucessfully using ISPC on Windows/MinGW64 ?
Some years ago I used to successfully compile & link ISPC programs (directly called from C++ without the ISPC runtime) with windows GNU toolchain based on MinGW64.
Now I upgraded ISPC to version 18 and the MinGW64. Compiling still works but the linker complains 

Warning: corrupt .drectve at end of def file 

whenever I try to link an ispc object-file.

**Comment by RoyiAvital** [comment]
See https://github.com/ispc/ispc/discussions/3313.
